CHANGELOG.md

```diff
@@ -50,7 +53,6 @@ compatible with the ink! `4.0.0` release.
 
 For full compatibility requirements see the [migration guide](https://use.ink/faq/migrating-from-ink-3-to-4/#compatibility).
 
-### Added
```

Comment:
Oh, why was this removed? Looks accidental @pmikolajczyk41 .

Reply:
I must have done this during some semi-automatic merging - that explains the last request for change above :|